trailblazer/store/api.py

```diff
@@ -26,33 +31,72 @@ def setup(self):
         new_info = self.Info()
         self.add_commit(new_info)
 
-    def find_analysis(self, family, started_at, status):
-        """Find a single analysis."""
-        query = self.Analysis.query.filter_by(family=family, started_at=started_at, status=status)
+    def info(self) -> models.Info:
+        """Return metadata entry."""
+        return self.Info.query.first()
+
+    def track_update(self):
+        """
+        used in CLI
+        Update the latest updated date in the database."""
+        metadata = self.info()
+        metadata.updated_at = dt.datetime.now()
+        self.commit()
+
+    def find_analysis(self, case_id, started_at, status):
```

Comment:
Should we rename this to
```suggestion
    def get_analysis(self, case_id, started_at, status):
```
as it actually returns an analysis

Reply:
Also it would be very nice with type hints and return values for these functions to make them more readable

Reply:
done